install/operator/pkg/cmd/internal/uninstall/uninstall_test.go

```diff
@@ -61,7 +61,7 @@ func TestUninstall(t *testing.T) {
 	// Create a fake client to mock API calls and pass it to the cmd
 	objs := []runtime.Object{&s}
 	cl := fake.NewFakeClient(objs...)
-	cl.List(ctx, client.InNamespace(ns), &sl)
+	cl.List(ctx, &sl)
```

Comment:
listing from a specific namespace has been removed, is this intentional? You can achieve the same as before with 
```
listOpts := []client.ListOption{
  client.InNamespace(ns),        
}
cl.List(ctx, &sl, listOpts...)
```

Reply:
Thanks!